cyclus/CMakeLists.txt

```
@@ -10,7 +10,7 @@ INCLUDE_DIRECTORIES(${CYCLUS_CORE_INCLUDE_DIRS})
 message(STATUS "Generating Type System API for Python")
 EXECUTE_PROCESS(COMMAND python ${CMAKE_CURRENT_SOURCE_DIR}/gentypesystem.py
     "--src-dir=${CMAKE_CURRENT_SOURCE_DIR}"
-    "--cyclus-version=${core_version}"
+    "--cyclus-version=${DATA_MODEL_VERSION}"
```

Do we want to change this argparse variable for `gentypesystem.py`?
```suggestion
    "--data-model-version=${DATA_MODEL_VERSION}"
```

---

Addressed this.  At some point (maybe not right now) should we rename `version::core()` or keep it as is?  As is, `version::core()` is the "source code version" and `version::describe()` is the "data model version"

---

Let's hold off on this for now